admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+# Extract metadata information from documents
+
+## Overview
+
+### Introduction
+
+The metadata extraction tool is designed to help you quickly identify, retrieve, and save key details from PDF documents, such as titles, dates, keywords, and other essential information. Powered by machine learning, this tool automates much of the extraction process, saving you time and making it easier to analyze large sets of documents. Whether you’re documenting case studies, managing witness statements, or reviewing legal reports, this tool simplifies data extraction, allowing you to focus on analysis and decision-making.
+
+### How it works
+
+The metadata extraction tool uses machine learning models trained on a vast collection of documents to recognize patterns and extract relevant information. It scans your documents, breaks down the content into smaller segments, like words or phrases, and uses the patterns it learned during training to identify which sections of the document contain useful information. The models learn to do this by being trained on large amounts of sample data, known as training data, which includes examples of documents with labeled information. This allows the models to recognize similar patterns in future documents.
+
+The effectiveness of this process is measured by accuracy, which refers to how often the models correctly identify and extract the right information. While the models strive for high accuracy, their performance can vary based on the quality of the documents and how closely they resemble the data the models were trained on. Over time, the models can be improved with more training data, enabling them to handle a broader range of document types and complex layouts.
```

Comment:
`their performance can vary based on the quality of the documents`

To me, the quality of the documents sounds a little dubious here without clarifications. I can imagine a reader asking if it refers to a merely technical quality, like the quality of the scan, or the quality in some other sense.